### PR TITLE
Add message.web_page attributes

### PR DESCRIPTION
**Pull request by mendelmaleh**

### CODE REVIEW

File: pyrogram/client/types/messages_and_media/webpage.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    """A webpage preview
+
+    Parameters:
+        id (``str``):
+            Unique identifier for this webpage.
+
+        url (``str``):
+            Full URL for this webpage.
+
+        display_url (``str``):
```
Comment (delivrance) [inline pyrogram/client/types/messages_and_media/webpage.py]: Required and optional arguments are like in the raw type, good.

File: pyrogram/client/types/messages_and_media/webpage.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+            Display URL for this webpage.
+
+        type (``str``, *optional*):
+            Type of webpage preview, known types (at the time of writing) are:
+            `article`, `photo`, `gif`, `video` and `document`,
+            `telegram_user`, `telegram_bot`, `telegram_channel`, `telegram_megagroup`.
```
Comment (delivrance) [inline pyrogram/client/types/messages_and_media/webpage.py]: While I strongly prefer having comprehensive docs, in this case we have no way of telling what all the possible values are (unless maybe we look at official clients/libs). This approach is good enough, except the style you used. Maybe it's better to use \*italic* instead, like I did in other places.

Comment (mendelmaleh) [reply pyrogram/client/types/messages_and_media/webpage.py]: f7bfd55

Comment (mendelmaleh) [reply pyrogram/client/types/messages_and_media/webpage.py]: Is this ok?

**Pull request merged**